Header header
float64 lid
float64 lambda
float64 tau_delta
float64 mu
float64 gamma
float64 angle_desired
uint8 curve
int32[] order_inv
